src/components/profile/Profile.js

```diff
@@ -13,11 +13,13 @@ import ViewAvatar from './ViewAvatar'
 
 const TITLE = 'Profile'
 
+const PrivateIcon = props => <IconButton {...props} wrapperStyle={styles.iconLeft} name="person-outline" />
+
 const EditIcon = props => <IconButton {...props} wrapperStyle={styles.iconRight} name="edit" />
 
 const IconButton = ({ onPress, disabled, wrapperStyle, ...iconProps }) => (
   <View style={[styles.icon, wrapperStyle]}>
-    <Icon onPress={onPress} raised color="rgb(85, 85, 85)" {...iconProps} />
+    <Icon onPress={onPress} color="#0C263D" {...iconProps} reverse size={20} />
```

Comment:
Can we wrap `IconButton` with `withStyles`, and use `theme.colors.darkBlue`?

Reply:
Done